File: cf_90_yet_another_integer_prob.py
test=int(input())
for i in range(test):
    a,b=map(int,input().split())
    if a==b:
        print(0)
    else:
        count=0
        diff=abs(a-b)
        if diff>=10:
            count=int(diff/10)
            diff=diff%10
        if diff==0:
            print(count)
        else:
            print(count+1)
        # for i in range(9):
        #     k=9-i
        #     while(diff>=k):
        #         count+=1
        #         diff-=k
        #         if diff==0:
        #             print(count)
        #             break

File: cf_90_yet_another_integer_prob.cpp
#include <iostream>
#include <cmath>
using namespace std;
int main(){
    ios_base::sync_with_stdio(false);cin.tie(NULL);cout.tie(NULL);
    int test;
    cin>>test;
    for(int i=0;i<test;i++){
        int a,b,diff;
        cin>>a>>b;
        if(a==b){
            cout<<0<<"\n";
        }
        else{
            int count=0;
        diff=abs(a-b);
        for(int i=0;i<10;i++){
           int  k=10-i;
            while(diff>=k){
                diff-=k;
                count+=1;
                if (diff==0)
                {
                    cout<<count<<"\n";
                    count=0;
                    break;
                }
            }
        }
        }
    }
        return 0;
}